cmd/root.go

```diff
@@ -45,7 +45,7 @@ func Execute() {
 
 	if err := RootCmd.Execute(); err != nil {
 		fmt.Println(err)
-		os.Exit(-1)
+		os.Exit(3)
```

Comment:
Hi Alex, thanks for contributing!

Why code 3? I believe that a better fitting would be 1 (https://www.thegeekstuff.com/2010/10/linux-error-codes/)

Please dont forget to sign the CLA